### ISSUE TITLE
Anchor links next to sections

### ISSUE THREAD

**Issue by maciej** [di-in-scala/di-in-scala.github.io#1]
Anchor links next to chapters and section titles would be nice.

Like the ones [here](http://lihaoyi.github.io/fastparse/#WritingParsers):
![screen shot 2015-07-03 at 10 27 37](https://cloud.githubusercontent.com/assets/9706/8495491/4f39211c-216e-11e5-8dab-b4bfa57426b5.png)

Without them one has to scroll to the ToC to get the anchor link. Also currently anchors links point only to chapters. No section links are available.

**Comment by adamw** [comment]
Do you know what tool does that OOTB? :)

**Comment by maciej** [comment]
No. fastparse documentation uses [Scalatex](https://github.com/lihaoyi/Scalatex), if that helps.

**Comment by adamw** [comment]
Ok thx I'll take a look one day. Would be awesome if generating documentation wouldn't be a big problem  with so many not-perfect choices one day ;)
